pom.xml

```diff
@@ -34,9 +34,9 @@
         <!-- This plugin key is to be used as a unique ID of the plugin incl.
         version and not as a key regarding the Atlassian plugin descriptor -->
         <plugin.key>${project.groupId}.${project.artifactId}-${project.version}</plugin.key>
-        <buildinfo.version>2.13.11</buildinfo.version>
+        <buildinfo.version>2.13.13</buildinfo.version>
         <buildinfo.maven.version>2.13.11</buildinfo.maven.version>
-        <buildinfo.gradle.version>4.9.9</buildinfo.gradle.version>
+        <buildinfo.gradle.version>4.10.0</buildinfo.gradle.version>
         <buildinfo.ivy.version>2.13.11</buildinfo.ivy.version>
```

Comment:
It is safe to upgrade **buildinfo.ivy.version** from *2.13.11* to *2.13.13*.

Reply:
done